reports/plain_vanilla_report_templates/get_MARCxml.rb

```diff
@@ -1,10 +1,88 @@
 require 'archivesspace/client'
 require 'active_support/all'
 require 'nokogiri'
-require_relative 'helper_methods.rb'
+require_relative '../../helper_methods.rb'
 
 aspace_staging_login()
 
-marc_record = @client.get("/repositories/4/resources/marc21/2065.xml")
-marc_document = Nokogiri::XML(marc_record.body)
-puts marc_document
+temp_file = "temp.xml"
+filename = "out.xml"
+
+resources = get_all_resource_uris_for_institution
+#remove this filter when testing is finished; I'm just testing with two records here
+resources[0..1].each do |resource|
```

Comment:
Did you want. to remove this before committing?

Reply:
Not just yet!